### PR TITLE
[AV-1835] Added key delete function

### PR DESCRIPTION
**Pull request by dasconnor**
Add a command to delete a key and a test.

### CODE REVIEW

File: cmd/keycmd/delete.go [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+import (
+	"errors"
+	"os"
+
+	"github.com/ava-labs/avalanche-cli/pkg/prompts"
+	"github.com/ava-labs/avalanche-cli/pkg/ux"
+	"github.com/spf13/cobra"
+)
+
+var forceDelete bool
```
Comment (felipemadero) [inline cmd/keycmd/delete.go:14]: maybe can just be called force. just a nit I know

**Review by holisticode — COMMENTED** [review]
No more comments other than those from @felipemadero

File: cmd/keycmd/delete.go [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+// avalanche key delete
+func newDeleteCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:          "delete",
+		Short:        "Delete a subnet configuration",
+		Long:         "The subnet delete command deletes an existing subnet configuration.",
```
[the remark this answers is not in the document]
Comment (dasconnor) [reply cmd/keycmd/delete.go]: fixed

File: cmd/keycmd/delete.go (hunk 1 above)
Comment (dasconnor) [reply cmd/keycmd/delete.go:14]: the reason I did this is there is also a `--force` flag for the `create` command. I think it's better to disambiguate, so I'm going to leave it as is for now, but could be convinced to consolidate flag variables later.

File: cmd/keycmd/delete.go [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+		Args:         cobra.ExactArgs(1),
+		SilenceUsage: true,
+	}
+	cmd.Flags().BoolVarP(
+		&forceDelete,
+		forceFlag,
+		"f",
+		false,
+		"overwrite an existing key with the same name",
+	)
```
[the remark this answers is not in the document]
Comment (dasconnor) [reply cmd/keycmd/delete.go:32]: clarified

**Review by holisticode — APPROVED** [review]

**Pull request merged**